### PR TITLE
test(CI): IE8 now requires a polyfill

### PR DESCRIPTION
**Pull request by vvo**
IE8 is dying, but the helper can still work if you include a polyfill.

Like this:

```html
<!--[if lte IE 8]>
<script src="https://cdn.jsdelivr.net/core-
js/2.0.0-alpha/shim.min.js"></script>
<![endif]-->
```

FTR I did not use https://cdn.polyfill.io/v2/docs/ because of a weird
"out of stack space" bug that I traced down to the Object.create
polyfill ONLY when running the tests (crazy browserify stuff
hapening).

I would say core-js seems to be working nicer and including it inside
a conditionnal comment should solve our issues.

I am looking forward simplifying our testing stack, it's currently
heavy with lot of code we do not need to be added (like tape
browserify streams etc..)

Also upgrade browserify, watchify, zuul and browzers (list of reusable browsers to test)

### CODE REVIEW

**Comment by vvo** [comment]
weeee passing PR!

**Comment by vvo** [comment]
Do not merge right now

File: test/run.js [hunk 1]
```diff
@@ -1,6 +1,6 @@
 'use strict';
 
-require('bulk-require')(__dirname, ['spec/**/*.js']);
+// require('bulk-require')(__dirname, ['spec/**/*.js']);
```
Comment (bobylito) [inline test/run.js]: Maybe not necessary to keep that? :)

**Comment by bobylito** [comment]
Looks good :) Thanks for taking the time to do that!

File: test/run.js (hunk 1 above)
Comment (vvo) [inline test/run.js]: Just doing a test to see whats wrong in microsoft edge (working locally though), will remove then

**Comment by vvo** [comment]
Ok edge is failing only because its an old version on saucelabs it seems, works in real life

**Pull request merged**